unity/Holo/Assets/Editor/ModelLoad/SingleModel.cs

```diff
@@ -84,8 +91,20 @@ protected void AddLayerComponent(GameObject go, ModelLayerInfo layerInfo)
         {
             ModelLayer layer = go.AddComponent<ModelLayer>();
             layer.Caption = layerInfo.Caption;
-            layer.Simulation = layerInfo.Simulation;
+            layer.Simulation = CheckIfSimulation(layerInfo.Simulation);

```

Comment:
here is a Simulation variable that contains a bool, computed from `Simulation` variable containing a string... more than a little confusing

Reply:
Same as above.